integration/config/configtx.yaml

```diff
@@ -1,4 +1,3 @@
-# Copyright Intel Corp. All Rights Reserved.
```

Comment:
elsewhere in this file is the field to edit for enabling raft
`OrdererType: solo` to `OrdererType: etcdraft`
There seems to be a lot of extra config in here that's not needed to run.

Reply:
The approach was to keep as close as possible to the sample config in the fabric repo so it would be easy to see what has changed and upgrade (i.e., a kind of 3-way merge  of `delta new vs old fabric config` + `delta fpc vs fabric config`.

So personally i would leave the somewhat redundant information and just make sure that when we instantiate the network in `fabric/bin/....` we make sure we select the raft profile